### ISSUE TITLE
mpd module crash if song contains "&"

### ISSUE THREAD

**Issue by tmpm697** [Alexays/Waybar#1172]
I've got this error also when the module crashed:
```
from markup due to error parsing markup: Error on line 1: Entity did not end with a semicolon;
most likely you used an ampersand character without intending to start an entity — escape ampersand as &amp;
```

my mpd module contains format:
```
"format": "{}"
```

**Comment by tmpm697** [comment]
Due to my mod config, not related to waybar.

**Comment by PeterNerlich** [comment]
I have the same problem, what was your fix? In #162 it is suggested to set `escape` to `true`, but the `mpd` module doesn't have this property.

**Comment by PeterNerlich** [comment]
@tmpm697 I have the same problem, what was your fix? In #162 it is suggested to set `escape` to `true`, but the `mpd` module doesn't have this property.

**Comment by tmpm697** [comment]
@PeterNerlich I don't remember, but my current `mpd.conf` contains these:
```
music_directory       "~/.cache/music"
playlist_directory    "~/.config/mpd/playlists"
db_file               "~/.config/mpd/database"

input {
        plugin "curl"
}

audio_output {
       type   "pulse"
       name   "Local Music Player Daemon"
}
```
